src/sentry/incidents/models/alert_rule.py

```diff
@@ -497,6 +497,7 @@ class AlertRuleTriggerAction(AbstractNotificationAction):
     status = BoundedPositiveIntegerField(
         default=ObjectStatus.ACTIVE, choices=ObjectStatus.as_choices()
     )
+    description = models.CharField(max_length=1000, null=True)
```

Comment:
Did we decide to do this per metric alert? From the ticket it seemed like we only really needed to support this for slack

Reply:
It's per action and yes it will be scoped to Slack only (we'll have to limit the UI)

Reply:
Just want to check if this description length makes sense for sending to slack, or will it be too long?

Reply:
If it's only for slack, is there a reason to not follow what we did for issue alerts? I could see a scenario where we want to support a description for another action, but the field is reserved for slack

Reply:
Right, should this be at the actual alert rule level, so it doesn't need to be configured for each action (assuming we add other actions later)?

Reply:
For metric alerts without slack actions, we'd also have this null description field which would potentially be confusing

Reply:
Issue alerts uses django forms so I was able to just pull the data from the form - metric alerts UI is built with React so I need to store this data somewhere. It won't limit a future for other actions to use the field, it's just only being used for Slack right now. Nothing about this field is Slack specific - I was just thinking we'd only display the textfield for Slack in the UI.

Reply:
I asked product if they wanted this per alert or per action and I was told per action.

Reply:
I'm not too familiar with metric alert actions (only issue alerts), but could we store this description in `AlertRuleTriggerAction` instead?

Reply:
That is where it's being stored

Reply:
oh whoops I misunderstood this whole time. I thought it was on the alert rule model 🙈

Reply:
was so confused there

Reply:
Hahah no problem, there are a lot of models in this file

Reply:
I guess final thing is just what dan mentioned above, the text limit which I have no clue on

Reply:
Oh yeah I replied outside the thread: https://github.com/getsentry/sentry/pull/72328#issuecomment-2155333814 but I checked that